presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -133,10 +151,15 @@ Array Functions
     Subsets array ``x`` starting from index ``start`` (or starting from the end
     if ``start`` is negative) with a length of ``length``.
 
-.. function:: transform(array, function) -> array
-    :noindex:
+.. function:: transform(array<T>, function<T,U>) -> array<U>
+
+    Returns an array that applies ``function`` to each element of ``array``::
```

Comment:
This makes it sounds as if the array was the one to apply the function

Reply:
Agreed, the original was bad, I'll improve it

Reply:
"Returns an array that is the result of applying ..."